tests/console/man_pages.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: man-pages
+# Summary: Basic functional test for man-pages
+# Maintainer: QE Core <qe-core@suse.de>
+
+use base 'consoletest';
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use strict;
+use warnings;
+use utils 'zypper_call';
+
+sub run {
+    my ($self) = @_;
+    select_serial_terminal;
+
+    if (script_run 'rpm -q man-pages man') {
```

Comment:
Is there any reason to add this test? 

if no, we can use single line `zypper_call 'in man-pages man';`

Reply:
Noted. I changed it.